Ginger/Ginger/Actions/ActionEditPage.xaml.cs

```diff
@@ -1048,23 +1041,23 @@ bool GetBooleanValue(string input)
                         switch (node.Title)
                         {
                             case "Description":
-                                node.IsSelected = GetBooleanValue(columnPreferencesArray[0]);
+                                node.IsSelected = string.IsNullOrEmpty( columnPreferencesArray[0])?false:true;
                                 break;
                             case "Path":
-                                node.IsSelected = GetBooleanValue(columnPreferencesArray[1]);
+                                node.IsSelected = string.IsNullOrEmpty(columnPreferencesArray[1]) ? false : true;
                                 break;
                             case "Actual Value":
-                                node.IsSelected = GetBooleanValue(columnPreferencesArray[2]);
+                                node.IsSelected = string.IsNullOrEmpty(columnPreferencesArray[2]) ? false : true;
                                 break;
                             case "Expected Value":
-                                node.IsSelected = GetBooleanValue(columnPreferencesArray[3]);
+                                node.IsSelected = string.IsNullOrEmpty(columnPreferencesArray[3]) ? false : true;
                                 break;
                             case "Store To":
-                                node.IsSelected = GetBooleanValue(columnPreferencesArray[4]);
+                                node.IsSelected = string.IsNullOrEmpty(columnPreferencesArray[4]) ? false : true;
```

Comment:
If store to is not in preference in user profile, this will line throw the error columnPreferencesArray[4]?

Reply:
no it will not because we are giving the default value.